Misc/NEWS.d/next/Core and Builtins/2021-12-06-15-32-12.bpo-42918.Czpgtg.rst

```diff
@@ -0,0 +1 @@
+Improve build-in :func:`compile` in mode 'single'.  Patch by Weipeng Hong.
```

Comment:
Could you please add more context on what this PR is fixing here? This current text is unfortunately too generic.